open_event/views/admin/models_views/event.py

```diff
@@ -146,7 +175,10 @@ def event_session_edit(self, event_id, session_id):
 
     @expose('/<event_id>/session/<session_id>/delete', methods=('GET', 'POST'))
     def event_session_delete(self, event_id, session_id):
-        DataManager.remove_session(session_id)
+        if is_event_owner(event_id):
```

Comment:
This repeated code is probably best refactored to be a decorator, such that you do

```
@requires_event_owner
def foo():
....
````

and then you only write this code once. We can refactor that in a future PR